### PR TITLE
Week 2

### PR DESCRIPTION
**Pull request by jolsenvaldez**
This is a pull request for my Week 2 branch into the master once I have a final version of my Week 2 Problem Set

### CODE REVIEW

**Comment by vaga3461** [comment]
Hi Juliana, should I open this file in R Studio or Jupyter?

**Comment by vaga3461** [comment]
> Hi Juliana, should I open this file in R Studio or Jupyter?

Never mind, realized I can see it in 'files changed'!

File: week2.Rmd [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+By the end of this exercise, you will be able to:
+1) Describe what albedo is, how it relates to emissivity, and how it can be substituted into the Stefan-Boltzmann Law equation. 
+2) Calculate the temperature of the earth given different values for albedo. 
+3) Use a basic plotting function and data to visualize quantitative values related to albedo. 
+
+##Background and Defining Terms
+Let's first re-familiarize ourselves with the Stefan-Boltzmann Law:
+$$ \begin{align} \frac {P}{A} = e \sigma T^4 \end{align}$$
+Terms and Units: 
+$P = Power\ (Watts)$
```
Comment (vaga3461) [inline week2.Rmd:43]: if you could put these in SI base units as well, that would be helpful! (Watts = kg m^2/s^3)

File: week2.Rmd [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+Let's first re-familiarize ourselves with the Stefan-Boltzmann Law:
+$$ \begin{align} \frac {P}{A} = e \sigma T^4 \end{align}$$
+Terms and Units: 
+$P = Power\ (Watts)$  
+$ A=Area\ (meters^2)$
+$e=emissivity\ (unitless)$
+$\sigma = Stephan-Boltzmann  \  constant\  (5.67x 10^{-8}\ W/m^2/T^4)$
+$ T= Temperature\ (degrees\  Kelvin^4)$
+
+The Stefan-Boltzmann equation creates a proportional relationship between energy ($\frac{W}{m^2}$) and temperature (kelvins) to the fourth power. compares an energy (power per unit area squared, with units in $\frac{W}{m^2}$) to temperature raised to the fourth (with units in $Kelvin^4$).
```
Comment (vaga3461) [inline week2.Rmd]: I think something got cut off before the word "compares"

File: week2.Rmd [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+
+We briefly touched on emissivity in the last problem set, but we'll be delving into that more this week. As a reminder, emissivity refers to the effectiveness of an object to emit an energy. For theoretical black bodies, which absorb all incident heat radiation, emissivity is equal to 1.
+
+**Albedo** is a term that refers to the amount of light that a planet (or any surface) reflects away. This makes albedo kind of like an "opposite" to emissivity. Therefore, a body that perfectly reflects all light (like one covered in snow or ice) would have an albedo of 100% (usually written in decimal form to be 1.00). In contrast, a planet that has a dark surface (or a black body) would have an albedo of 0. And just like emissivity, albedo is unitless. 
+
+We relate albedo to emissivity through the equation: 
+$$ e = 1 - albedo $$
+Let's stop and think about if this equation makes sense -> 
+-We expect a blackbody to have an albedo of zero and an emissivity of 1 since all light is absorbed and none is reflected. e = 1-0, e = 1. Great!
```
Comment (vaga3461) [inline week2.Rmd:59]: love this quick "makes sense" check

File: week2.Rmd [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+-We expect a blackbody to have an albedo of zero and an emissivity of 1 since all light is absorbed and none is reflected. e = 1-0, e = 1. Great! 
+-We expect a perfect reflector to have an albedo of one and an emissivity of zero since no light is absorbed and all is reflected. e = 1-1, e = 0. Awesome! 
+
+Now, let's rewrite the Stefan-Boltzmann Law to include albedo: 
+$$ \frac {P}{A} = (1-albedo)\ \sigma T^4 $$
+We'll be using albedo to calculate a more accurate temperature for the earth, plot how different albedos affect Earth's temperature, and think about Earth's climate. 
+
+##Example
+First, we'll start by using albedo to explore the difference in temperature on Mars if it were mostly covered in ice (some scientists think it was once an icy planet!) versus in its current red, sandy state. We'll be using a version of the Stefan-Boltzmann law where we'll solve for Temperature, but this week we'll substitute our $1-albedo$ term in for the emissivity. Here's what the initial equilibrium equation looks like: 
+$$(1-albedo)*So*\pi R^2 = \sigma T^4*4\pi R^2$$
```
Comment (vaga3461) [inline week2.Rmd:68]: I'm not sure So is defined anywhere. That could use a refresher, even if it was included in last week's lab.

File: week2.Rmd [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+This equation says that the radiation received by a planet (now affected by the albedo of that planet) is equal to the radiation emitted by that planet. 
+
+Here is the equation above solved for temperature (kelvins):
+$$T = [(1-albedo)*So/4\sigma]^{1/4}$$
+We'll use a solar constant for Mars of 591 $\frac {W}{m^2}$, the Stefan Boltzmann constant of $5.67x 10^{-8}$ and albedos of 0.70 for icy Mars and 0.15 for modern Mars.
+
+This week, you'll be introduced to a new coding tool that makes it easy to define and plot data. We'll first use tibbles to create a data frame with columns and rows.
```
Comment (vaga3461) [inline week2.Rmd]: maybe rephrase as "...you'll be introduced to a new coding tool, called a tibble, that makes..."

Comment (vaga3461) [reply week2.Rmd]: or is the new tool ggplot? Either way, you could mention it here :)

**Review by vaga3461 — COMMENTED** [review]
Overall this looks really good! Just a few minor points to clarify terms and units.

**Review by MzCorona — COMMENTED** [review]
I am having a hard time finding your "ReviewNB" link... I may comment on GitHub...

File: week2.Rmd [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+library(ggplot2)
+```
+
+##Introduction
+Last week, we introduced the Stefan-Boltzmann Law and applied it to questions regarding astronomical energies and temperatures, specifically with respect to black bodies. This week, we'll build on the Stefan-Boltzmann Law to include terms like albedo and calculations related to climate. 
+
+##Learning Goals
+By the end of this exercise, you will be able to:
+1) Describe what albedo is, how it relates to emissivity, and how it can be substituted into the Stefan-Boltzmann Law equation. 
+2) Calculate the temperature of the earth given different values for albedo.
```
Comment (MzCorona) [inline week2.Rmd]: "Earth" should be capitalized ^_^

File: week2.Rmd [hunk 7]
```diff
@@ -0,0 +1,138 @@
...
+```
+
+##Introduction
+Last week, we introduced the Stefan-Boltzmann Law and applied it to questions regarding astronomical energies and temperatures, specifically with respect to black bodies. This week, we'll build on the Stefan-Boltzmann Law to include terms like albedo and calculations related to climate. 
+
+##Learning Goals
+By the end of this exercise, you will be able to:
+1) Describe what albedo is, how it relates to emissivity, and how it can be substituted into the Stefan-Boltzmann Law equation. 
+2) Calculate the temperature of the earth given different values for albedo. 
+3) Use a basic plotting function and data to visualize quantitative values related to albedo.
```
Comment (MzCorona) [inline week2.Rmd]: What are some types of basic plotting functions?

File: week2.Rmd (hunk 3 above)
Comment (MzCorona) [reply week2.Rmd:59]: Yay definitions!

File: week2.Rmd [hunk 8]
```diff
@@ -0,0 +1,138 @@
...
+$$(1-albedo)*So*\pi R^2 = \sigma T^4*4\pi R^2$$
+This equation says that the radiation received by a planet (now affected by the albedo of that planet) is equal to the radiation emitted by that planet.
+Note that we are using So, which is the solar constant ($W/m^2$) and describes the amount of energy that the earth receives from the sun. 
+
+Here is the equation above solved for temperature (kelvins):
+$$T = [(1-albedo)*So/4\sigma]^{1/4}$$
+We'll use a solar constant for Mars of 591 $\frac {W}{m^2}$ (this would just describe the energy received by Mars from the sun), the Stefan Boltzmann constant of $5.67x 10^{-8}$ and albedos of 0.70 for icy Mars and 0.15 for modern, sandy Mars.
+
+This week, you'll be introduced to some new coding terms:
+-tibble is a function that creates data frames which we can populate with values and make calculations from
```
Comment (MzCorona) [inline week2.Rmd:77]: As a student, I like definitions in problem sets

File: week2.Rmd [hunk 9]
```diff
@@ -0,0 +1,138 @@
...
+   b) add a range of albedos (), Stefan-Boltzmann constant (SBc),  and a calculated temperature column for the data frame. 
+   c) use the data frame to plot albedo versus temperature for the earth.
+   d) comment on the trend of the ggplot you created. Do the different albedos change the Temperature by much? Is the trend linear?
+Hint: look back to the example for all the code you'll need. Simply rename columns and data frames to reflect information for the earth instead of Mars!
+ 
+_________________________________________________________
+
+4) After completing questions 2 and 3, what do you think about the temperatures that you calculated? Do they seem too hot or too cold? If they weren't what you were expecting given the temerature of Earth today, what could be missing from the model that we considered in calculating the earth's temperature? 
+
+_________________________________________________________
```
Comment (MzCorona) [inline week2.Rmd:130]: I really like the directions you give with the questions. You give enough guidance but not too much. That's always a tricky balance to find. Nice job!

**Review by MzCorona — COMMENTED** [review]
Okay I reviewed on GitHub. I think Vanessa hit all the points well. Overall great job!
